pkg/apis/provisioning/v1alpha5/register.go

```diff
@@ -41,7 +41,7 @@ var (
 		return nil
 	})
 	ProvisionerNameLabelKey         = Group + "/provisioner-name"
-	NotReadyTaintKey                = Group + "/not-ready"
```

Comment:
We can't place a taint on the node (or at least would need to remove it fairly quickly) as it prevents daemonsets from binding so we don't get any device plugins and extended resources never register.